pkg/export/bench/app/example_app.go

```diff
@@ -77,65 +77,79 @@ var (
 	metricIncomingRequestsPending = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "example_incoming_requests_pending",
+			Help: "Number of incoming requests currently pending.",
 		},
 		[]string{"status", "method", "path"},
 	)
 	metricOutgoingRequestsPending = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Name: "example_outgoing_requests_pending",
+			Help: "Number of outgoing requests currently pending.",
 		},
 		[]string{"status", "method", "path"},
 	)
 
 	metricIncomingRequests = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "example_incoming_requests_total",
+			Help: "Total number of incoming requests.",
 		},
 		[]string{"status", "method", "path"},
 	)
 	metricOutgoingRequests = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "example_outgoing_requests_total",
+			Help: "Total number of outgoing requests.",
 		},
 		[]string{"status", "method", "path"},
 	)
 	metricIncomingRequestErrors = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "example_incoming_request_errors_total",
+			Help: "Total number of errors on incoming requests.",
 		},
 		[]string{"status", "method", "path"},
 	)
 	metricOutgoingRequestErrors = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "example_outgoing_request_errors_total",
+			Help: "Total number of errors on outgoing requests.",
 		},
 		[]string{"status", "method", "path"},
 	)
 
 	metricIncomingRequestDurationHistogram = prometheus.NewHistogramVec(
+		//nolint:promlinter
```

Comment:
The lint is:
`pkg/export/bench/app/example_app.go|122 col 3| Metric: example_histogram_incoming_request_duration Error: metric name should not include type 'histogram' (promlinter)`

I wasn't sure if renaming would break anything else and didn't want to get too far sidetracked. I will just note that they are test metrics.

Reply:
SGTM!